frontend/src/tests/instructor-permission-test.js

```diff
@@ -238,10 +189,10 @@ export function instructorsPermissionTests(api) {
 
     it("can't update instructors except for self (i.e. active user)", async () => {
         // we use the default user's admin privilege here to fetch an unrelated instructor
-        let respFetchInst = await apiGET(`/instructor/instructors`);
+        const respFetchInst = await apiGET(`/instructor/instructors`);
```

Comment:
`let resp = await ...` and reuse `resp` unless you need to keep the response object around for some reason...